docs/guides/serverless-shared-migration-guide.md

```diff
@@ -0,0 +1,203 @@
+---
+page_title: "Migration Guide: Transition out of Serverless Instances and Shared-tier clusters"
+---
+
```

Comment:
Noting that migration guide for Flex-to-Dedicated will be included as a part of [CLOUDP-278908](https://jira.mongodb.org/browse/CLOUDP-278908), as noted [here](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2744/files#r1820614013).